sdk/storage/Azure.Storage.Common/src/Shared/ClientsideEncryption/Models/EncryptionData.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.Cryptography;
+using Metadata = System.Collections.Generic.IDictionary<string, string>;
+
+namespace Azure.Storage.Cryptography.Models
+{
+    /// <summary>
+    /// Represents the encryption data that is stored on the service.
+    /// </summary>
+    internal class EncryptionData
+    {
+        /// <summary>
+        /// The blob encryption mode.
+        /// </summary>
+        public string EncryptionMode { get; set; }
```

Comment:
Nit - should things like this be settable since they're effectively constants in v1?

Reply:
It's part of the JSON schema. We need to be able to parse the value and verify it.